### PR TITLE
Add badge for preferred citations

### PR DESCRIPTION
**Pull request by cthoyt**
This PR adds a "preferred" citation field as suggested in https://github.com/OBOFoundry/OBOFoundry.github.io/pull/2063#issuecomment-1231327188 and adds a nice display badge to match it so we can remove more redundant information from ECO and FBbt. It adds a test to make sure at most only one citation is marked as preferred.

This PR also removes some redundant information from ECO and FBbt that does not affect the overall meaning of the page.

Screen Shot 2022-08-30 at 11 15 20

### CODE REVIEW

File: ontology/fbbt.md [hunk 1]
```diff
@@ -65,8 +66,3 @@ usages:
   user: http://flybase.org
 activity_status: active
 ---
-
-An ontology representing the gross anatomy of Drosophila melanogaster.
```
Comment (cthoyt) [inline ontology/fbbt.md:69]: This text is removed because it is an exact duplicate of the description field.

**Review by matentzn — APPROVED** [review]
I wholeheartedly approve! THANK YOU!

- [ ] @Clare72 
- [ ] @mgiglio99 

please approve the changes to your metadata (now it appears lossless to me).

**Comment by Clare72** [comment]
I am not able to tick the box - but as I said on the other PR, I am happy with this solution.

**Review by matentzn — APPROVED** [review]
As a senior member of the TWG, I judge the change to eco.md to be one of presentation only, and therefore falls under the new side laws in the metadata change SOP: #2065

**Comment by mgiglio99** [comment]
Hello,
Sorry for the delayed reply. I trust your judgement that your changes are to presentation  only.  So it's fine with me.
Looking at this issue prodded me to review the ECO content on the OBO Foundry page and I realize we need to do some updating. We are going to work on some updates and make a pull request. Is that the correct path to take?
Thanks,
Michelle

**Comment by matentzn** [comment]
Thats right! Just make a pr on the file!

**Pull request merged**